src/screens/SendToken/SendTokenTransaction.js

```diff
@@ -92,28 +96,27 @@ const getTransactionSuccessTitle = (props) => {
   return t('transactions.title.transactionSuccess');
 };
 
-const CancelText = styled(MediumText)`
-  color: ${themedColors.negative};
-  font-size: ${fontSizes.medium}px;
-`;
-
-const ButtonWrapper = styled.View`
-  width: 100%;
-  margin: 0px ${spacing.layoutSides}px 20px;
-`;
+function SendTokenTransaction(props: Props) {
+  const dispatch = useDispatch();
+  const { navigation } = props;
+  const {
+    isSuccess,
+    error,
+    transactionPayload: { tokenType: transactionTokenType, extra: { allowance = {} } = {}, chain },
+    transactionType,
+    goBackDismiss,
+    noRetry,
+    batchHash = '',
+    hash = '',
+    accounts,
+  } = navigation.state.params;
```

Comment:
Wdyt of breaking this down to every param separately and with added type plus getting it through `useNavigationParam`?